validator/src/main.rs

```diff
@@ -761,11 +761,7 @@ pub fn main() {
     };
 
     if let Some(ref cluster_entrypoint) = cluster_entrypoint {
-        let udp_sockets = [
-            &node.sockets.gossip,
-            &node.sockets.broadcast,
-            &node.sockets.repair,
-        ];
+        let udp_sockets = vec![&node.sockets.gossip, &node.sockets.repair];
```

Comment:
It sorta sucks to remove the broadcast sockets from this check, as this test has been super useful in detecting misconfigured validator ports and reducing our support burden.  I have no idea why you're having trouble with this check though